uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -23,7 +23,8 @@ class DrupalUserServiceTest extends CRM_HRCore_Test_BaseHeadlessTest {
   private $testContact;
 
   public function setUp() {
-    $this->testContact = ContactFabricator::fabricate();
+    CRM_Utils_API_ReloadOption::singleton();
+    $this->loadRequiredData();
```

Comment:
The transaction rollback is enough to remove the data. I had problems where rollbacks failed before, but found that this was due to working with custom data. It's mentioned [here](https://wiki.civicrm.org/confluence/display/CRMDOC/Transaction+Reference) in the section "Special Topics: TRUNCATE and ALTER force immediate commit"